addons/website/static/src/systray_items/website_switcher.js

```diff
@@ -11,19 +12,48 @@ const { Component } = owl;
 export class WebsiteSwitcherSystray extends Component {
     setup() {
         this.websiteService = useService('website');
+        this.notificationService = useService("notification");
+        this.actionService = useService("action");
+        this.noDomainTooltip = this.env._t("This website does not have a domain set.");
     }
 
     getElements() {
         return this.websiteService.websites.map((website) => ({
             name: website.name,
             id: website.id,
+            noDomain: !website.domain,
```

Comment:
Nitpicking/Can ignore (just set as resolved if you disagree): I would have just added the domain in `domain` property so we can retrieve it should we need it later, rather than casting it to a boolean.
And so in the template, instead of `<t t-if="element.noDomain">` -> `<t t-if="!element.domain">`